frame/contracts/src/exec.rs

```diff
@@ -1955,29 +2051,33 @@ mod tests {
 		});
 
 		// This one tests passing the input data into a contract via instantiate.
-		ExtBuilder::default().build().execute_with(|| {
-			let schedule = <Test as Config>::Schedule::get();
-			let min_balance = <Test as Config>::Currency::minimum_balance();
-			let mut gas_meter = GasMeter::<Test>::new(GAS_LIMIT);
-			let executable = MockExecutable::from_storage(input_data_ch, &mut gas_meter).unwrap();
-			set_balance(&ALICE, min_balance * 10_000);
-			let contract_origin = Origin::from_account_id(ALICE);
-			let mut storage_meter =
-				storage::meter::Meter::new(&contract_origin, None, min_balance).unwrap();
-
-			let result = MockStack::run_instantiate(
-				ALICE,
-				executable,
-				&mut gas_meter,
-				&mut storage_meter,
-				&schedule,
-				min_balance,
-				vec![1, 2, 3, 4],
-				&[],
-				None,
-			);
-			assert_matches!(result, Ok(_));
-		});
+		ExtBuilder::default()
+			.with_code_hashes(MockLoader::code_hashes())
```

Comment:
Now that we take a deposit from the original code_hash upload, we need to have these hashes on chain or the tests will fail. This is what this with_code_hashes function does, and the reason for the changes, here, all the diff are mostly related to code formatting